src/game.cpp

```diff
@@ -875,22 +879,22 @@ bool game::start_game()
     get_safemode().clear_character_rules();
 
     //Put some NPCs in there!
-    if( get_option<std::string>( "STARTING_NPC" ) == "always" ||
-        ( get_option<std::string>( "STARTING_NPC" ) == "scenario" &&
+    if( get_option<std::string> ( "STARTING_NPC" ) == "always" ||
+        ( get_option<std::string> ( "STARTING_NPC" ) == "scenario" &&
           !g->scen->has_flag( "LONE_START" ) ) ) {
         create_starting_npcs();
     }
     //Load NPCs. Set nearby npcs to active.
     load_npcs();
     // Spawn the monsters
     const bool spawn_near =
-        get_option<bool>( "BLACK_ROAD" ) || g->scen->has_flag( "SUR_START" );
+        get_option<bool> ( "BLACK_ROAD" ) || g->scen->has_flag( "SUR_START" );
     // Surrounded start ones
     if( spawn_near ) {
         start_loc.surround_with_monsters( omtstart, mongroup_id( "GROUP_ZOMBIE" ), 70 );
     }
 
-    m.spawn_monsters( !spawn_near ); // Static monsters
+    m.spawn_monsters( !spawn_near );  // Static monsters
```

Comment:
Other unrelated whitespace changes are like this one.

Reply:
same thing, i have not altered this specific line myself. it's the automated 'make astyle' command.
i'm afraid i cannot comply with your requests since i must use 'make astyle' to pass the gorgon-ghprb checks..